website/pages/en/index.js

```diff
@@ -0,0 +1,154 @@
+const MarkdownBlock = CompLibrary.MarkdownBlock; /* Used to read markdown */
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
+
+const siteConfig = require(process.cwd() + "/siteConfig.js");
+
+function imgUrl(img) {
+  return siteConfig.baseUrl + "img/" + img;
+}
+
+function docUrl(doc, language) {
+  return siteConfig.baseUrl + "docs/" + (language ? language + "/" : "") + doc;
+}
+
+function pageUrl(page, language) {
+  return siteConfig.baseUrl + (language ? language + "/" : "") + page;
+}
+
+class Button extends React.Component {
+  render() {
+    return (
+      <div className="pluginWrapper buttonWrapper">
+        <a className="button" href={this.props.href} target={this.props.target}>
+          {this.props.children}
+        </a>
+      </div>
+    );
+  }
+}
+
+Button.defaultProps = {
+  target: "_self"
+};
+
+const SplashContainer = props => (
+  <div className="homeContainer">
+    <div className="homeSplashFade">
+      <div className="wrapper homeWrapper">{props.children}</div>
+    </div>
+  </div>
+);
+
+const Logo = props => (
+  <div className="projectLogo">
+    <img src={props.img_src} />
+  </div>
+);
+
+const ProjectTitle = props => (
+  <h2 className="projectTitle">
+    {siteConfig.title}
+    <small>{siteConfig.tagline}</small>
+  </h2>
+);
+
+const PromoSection = props => (
+  <div className="section promoSection">
+    <div className="promoRow">
+      <div className="pluginRowBlock">{props.children}</div>
+    </div>
+  </div>
+);
+
+class HomeSplash extends React.Component {
+  render() {
+    let language = this.props.language || "";
+    return (
+      <SplashContainer>
+        {/* <Logo img_src={imgUrl('scalameta-logo.png')} /> */}
+        <div className="inner">
+          <ProjectTitle />
+          <PromoSection>
+            <Button href={docUrl("users/installation.html", language)}>
+              Documentation
+            </Button>
+            <Button href={siteConfig.repoUrl} target="_blank">
+              View on GitHub
+            </Button>
+          </PromoSection>
+        </div>
+      </SplashContainer>
+    );
+  }
+}
+
+const Block = props => (
+  <Container
+    padding={["bottom", "top"]}
+    id={props.id}
+    background={props.background}
+  >
+    <GridBlock align="left" contents={props.children} layout={props.layout} />
+  </Container>
+);
+
+const Features = props => {
+  const features = [
+    {
+      title: "Remove dead code",
+      content: "Automatically fix common problems like remove unused imports.",
+      image: imgUrl("rewriting.png"),
+      imageAlign: "right"
+    },
+    {
+      title: "Catch bugs",
+      content: "Automatically enforce coding conventions of your organization.",
+      image: imgUrl("linting.png"),
+      imageAlign: "left"
+    },
+    {
+      title: "Extensible",
+      content: "Extend Scalafix with custom rewrite and linting rules",
+      image: imgUrl("extensible.png"),
```

Comment:
I would prefer we avoid using image for this feature. We cannot copy paste, it's not super accessible. I can do a followup PR for this.

Reply:
Let's improve the landing page in a followup PR